README.md

```diff
@@ -9,16 +9,40 @@ maintained by Interchain GmbH,
 and one in [Rust](https://github.com/informalsystems/tendermint-rs),
 maintained by Informal Systems.
 
-There have been inadvertent divergences in the specs followed
-by the Go implementation and the Rust implementation respectively.
-However, we are working toward reconverging these specs into a single unified spec.
-Consequently, this repository is in a bit of a state of flux.
+### Data Structures
 
-At the moment, the spec followed by the Go implementation
-(tendermint/tendermint) is in the [spec](spec) directory,
-while the spec followed by the Rust implementation
-(informalsystems/tendermint-rs) is in the [rust-spec](rust-spec)
-directory. TLA+ specifications are also in the rust-spec directory.
+- [Encoding and Digests](./spec/core/encoding.md)
+- [Blockchain](./spec/core/data_structures.md)
+- [State](./spec/core/state.md)
 
-Over time, these specs will converge in the spec directory.
-Once they have fully converged, we will version the spec moving forward.
+### Consensus Protocol
+
+- [Consensus Algorithm](./spec/consensus/consensus.md)
+- [Creating a proposal](./spec/consensus/creating-proposal.md)
+- [Time](./spec/consensus/bft-time.md)
+- [Light-Client](./spec/consensus/light-client/README.md)
```

Comment:
We have `spec/light-client` and `spec/consensus/light-client`. Is there a plan to merge these different sections?

Reply:
Yea, I would hope so.